webserver/views/api/v1/core.py

```diff
@@ -197,3 +197,39 @@ def get_many_lowlevel():
 
     return jsonify(recording_details)
 
+
+@bp_core.route("/count", methods=["GET"])
+@crossdomain()
+def get_many_count():
+    """Get low-level count for many recordings at once.
+
+    **Example response**:
+
+    .. sourcecode:: json
+
+       {"mbid1": 0,
+        "mbid2": 1
+       }
+
+    :query recording_ids: *Required.* A list of recording MBIDs to retrieve
+
+      Takes the form `mbid;mbid`.
+
+    :resheader Content-Type: *application/json*
+    """
+    recording_ids = request.args.get("recording_ids")
+
+    if not recording_ids:
+        raise webserver.views.api.exceptions.APIBadRequest(
+            "Missing `recording_ids` parameter")
+
+    recordings = _parse_bulk_params(recording_ids)
+    if len(recordings) > 200:
+        raise webserver.views.api.exceptions.APIBadRequest(
+            "More than 200 recordings not allowed per request")
+
+    mbids = set(r[0] for r in recordings)
+    recording_count = {}.fromkeys(mbids, 0)
```

Comment:
This is neat, I didn't know of this method before. Is the idea here to make sure that all MBIDs have a value in the returned dictionary even if it's 0?

This ties in to https://github.com/metabrainz/acousticbrainz-server/pull/223 a little bit too - we have to decide what to return for missing data.
I would be just as happy to return no data for mbids which are not in the database. In this case it's up to the caller to check if the keys exist.